### PR TITLE
check for positive integer

### PR DESCRIPTION
**Pull request by edsilv**

### CODE REVIEW

File: src/lib/embed.js [hunk 1]
```diff
@@ -154,8 +155,9 @@ docReady(function() {
             return 0;
         }
 
-        // only load jQuery if not already included in page.
-        if (!(j = window.jQuery) || compareVersionNumbers(version, j.fn.jquery) || callback(j, scriptUri, absScriptUri, loaded)) {
+        // load jQuery if not already included in page or
+        // if the version we want (1.10.2) is greater than what's available
+        if (!(j = window.jQuery) || isPositiveInteger(compareVersionNumbers(version, j.fn.jquery)) || callback(j, scriptUri, absScriptUri, loaded)) {
```
Comment (jcoyne) [inline src/lib/embed.js]: I would find this code easier to read if the function was more descriptive. e.g. 
```javascript
if(!correctVersionOfJqueryLoaded()) {
  dynamicallyLoadJquery();
}
```

Comment (jcoyne) [inline src/lib/embed.js]: It's still unclear to me what this third term is expected to return:
```
callback(j, scriptUri, absScriptUri, loaded)
```
is it ever anything besides `undefined`?

Comment (jcoyne) [inline src/lib/embed.js]: It does look like this code will fix the issue mentioned.

Comment (jcoyne) [inline src/lib/embed.js]: Actually, shouldn't it accept `0` too?  That would mean you already have a copy of `1.10.2` loaded.

**Comment by jcoyne** [comment]
I'm not even sure why we need this version check, because app.html pulls in another copy of jquery: https://github.com/UniversalViewer/universalviewer/blob/master/src/app.html#L6

**Comment by edsilv** [comment]
See: https://github.com/UniversalViewer/universalviewer/pull/475/commits/ff6d886953cbb85e5ffaaad31ab43e9f41e532dd

Have tried to make it a bit more readable, and included a check for a 0 value.

Not sure what's going on with that callback parameter. It's from an example I found approx 4 years ago when starting work on the wellcome player. embed.js is entirely phased out in v3.

**Pull request merged**